services/core-web/src/components/Forms/incidents/IncidentForm.tsx

```diff
@@ -397,6 +394,10 @@ const renderDocumentation = (childProps, isEditMode, handlers, parentHandlers) =
     (doc) => doc.mine_incident_document_type_code === "FIN"
   );
 
+  const handleRemoveDocument = (event, documentGuid: string) => {
```

Comment:
This is really the only material change to this file.  The rest was just a typescript upgrade and formatting.